### PR TITLE
Test colon dropstored methods

### PR DESCRIPTION
**Pull request by kshyatt**
These methods [aren't tested](https://codecov.io/gh/JuliaLang/julia/src/master/stdlib/SparseArrays/src/sparsematrix.jl#L3134) ... yet

### CODE REVIEW

File: stdlib/SparseArrays/test/sparse.jl [hunk 1]
```diff
@@ -1202,6 +1202,13 @@ end
     # --> Test dropping a block of the matrix towards the upper left
     SparseArrays.dropstored!(A, 2:5, 2:5)
     @test nnz(A) == 42
+    # --> Test dropping all elemeents
```
Comment (dkarrasch) [inline stdlib/SparseArrays/test/sparse.jl]: -> elements

**Review by dkarrasch — APPROVED** [review]
👍

File: stdlib/SparseArrays/test/sparse.jl (hunk 1 above)
Comment (kshyatt) [reply stdlib/SparseArrays/test/sparse.jl]: Fixed

**Comment by kshyatt** [comment]
Checked the logs, all the failures are unrelated. Good to go?

**Comment by StefanKarpinski** [comment]
Yikes, that's a lot of failures though. Let me restart a few.

**Comment by dkarrasch** [comment]
The last failing test is clearly unrelated
```julia
Error in testset Test:
Error During Test at D:\buildbot\worker-tabularasa\tester_win64\build\share\julia\stdlib\v1.5\Test\test\runtests.jl:863
  Got exception outside of a @test
  SystemError: opening file "C:\\cygwin\\tmp\\jl_7DA2.tmp": No such file or directory
```
so I'm merging this.

**Pull request merged**